src/zeit/content/article/edit/interfaces.py

```diff
@@ -411,6 +412,20 @@ class IAuthor(IReference):
         required=False)
 
 
+class IVolume(IReference):
+
+    references = zeit.cms.content.interfaces.ReferenceField(
+        title=_("Volume"),
+        description=_("Drag a volume here"),
+        source=zeit.content.volume.interfaces.VOLUME_SOURCE,
+        required=False)
+
+    teaserText = zope.schema.Text(
+        title=_("Teaser text"),
+        required=False,
+        max_length=170)
```

Comment:
Wir haben "lokales Überschreiben" z.B. bei Bildern die Caption bisher so gebaut, dass das auf dem Referenzobjekt stattfindet. Das heißt, das Artikelmodul weiß davon nix, d.h. das hat hier in seinem Interface auch nichts zu tun.